config/initializers/rack_attack.rb

```diff
@@ -55,26 +56,18 @@ def anonymous?
   end
 
   def authenticated?
-    warden_user.present?
-  end
-
-  protected
-
-  WARDEN_CHECK_EXCLUDE_URLS = ['/hmis/app_settings', '/hmis/user', '/messages/poll'].to_set.freeze
```

Comment:
note, we no longer need special handling since the auth check shouldn't interact with timeoutable's last_request_at